main_app/templates/main_app/create_contact.html

```diff
@@ -9,26 +9,56 @@
 	scroll-behavior:smooth;
```

Comment:
1. Reduce the width of the form.
2. Reduce the width of the select box as well.